config/stack.go

```diff
@@ -0,0 +1,97 @@
+package config
+
+import (
+	"context"
+
+	"github.com/zclconf/go-cty/cty"
+
+	"github.com/gruntwork-io/terragrunt/config/hclparse"
+	"github.com/gruntwork-io/terragrunt/internal/errors"
+	"github.com/gruntwork-io/terragrunt/options"
+)
+
+// StackConfigFile represents the structure of terragrunt.stack.hcl stack file
+type StackConfigFile struct {
+	Locals *terragruntLocal `cty:"locals"  hcl:"locals,block"`
+	Units  []*Unit          `cty:"unit" hcl:"unit,block"`
```

Comment:
We should point out in the PR body that we're not supporting recursive stacks yet. It should also make its way to the release notes.

Reply:
Updated description